src/content/caseStudies/contenda-share-pear/index.mdx

```diff
@@ -0,0 +1,81 @@
+
+Grady Salzman, software engineer, says, “Depending on your team size, Contenda could enable others to make content for you. You could have someone who is spending all their time and effort on recording videos. Someone else could take those videos and write a blog post in their own thoughts, tone, and voice. If you’re a smaller team, it could enable you to release faster.”
+
+When Contenda needed to share draft blog posts publicly with other people, they turned to Astro and Netlify as their solution. This combination of technologies let them stay laser focused on their goal of making collaboration and content creation easy.
+
+</div>
+
+<div class="my-8">
+	<ImageFigure caption="Take any video and turn it into effective written content with Contenda.">
+		<BlogContentImage src={videosToContent} />
+	</ImageFigure>
+</div>
+
+## Challenges
+
+Contenda is a team of less than 10 people. They often need to share blogs with each other in order to address pain points and conduct reviews but they couldn’t do these things without sharing their emails and API keys with one another. This also posed a security problem for customers that they knew they had to solve eventually. To resolve these issues for internal and external users, Contenda created [Share Pear](https://github.com/Contenda-Team/share-pear), an open-source tool for quickly sharing Markdown blog post previews.
+
+## Requirements
+
+[Share Pear](https://github.com/Contenda-Team/share-pear) needed to be built without requiring backend changes since Contenda did not want to overload themselves by managing additional complexity. Ideally, it also needed to be cost-effective but preferably free. Fortunately for the team, they were able to assess and select Astro as their solution all in a day’s time thanks to quick work from Cassidy Williams.
+
+Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.
+
+Contenda also knew they had to use Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.
```

Comment:
```suggestion
Cassidy (Contenda CTO) saw an opportunity to pair Astro with Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.
```

Reply:
Approved ✅